styles/prosilver/template/skeleton_body.html

```diff
@@ -18,7 +18,7 @@ <h2>{{ lang('PHPBB_CREATE_SKELETON_EXT') }}</h2>
 
 <div class="panel bg1">
 	<div class="inner">
-		<p class="rightside"><a href="https://area51.phpbb.com/docs/dev/31x/extensions/skeleton_extension.html" target="_blank"><i class="icon fa-book fa-fw" aria-hidden="true"></i><span>{{ lang('PHPBB_SKELETON_EXT_DOCS') }}</span></a></p>
+		<p class="rightside"><a href="https://area51.phpbb.com/docs/dev/32x/extensions/skeleton_extension.html" target="_blank"><i class="icon fa-book fa-fw" aria-hidden="true"></i><span>{{ lang('PHPBB_SKELETON_EXT_DOCS') }}</span></a></p>
 		<h2 class="solo">{{ lang('SKELETON_TITLE_EXTENSION_INFO') }}</h2>
 		{% for extension in loops.extension %}
```

Comment:
The `loops.` prefix is only needed in phpBB 3.1.x, since the skeleton will not be installable using it, I am wondering if it is worth to change these occurencies?

Reply:
It doesn't hurt to have them